system/lib/libcxxabi/src/private_typeinfo.cpp

```diff
@@ -806,7 +806,7 @@ __dynamic_cast(const void *static_ptr, const __class_type_info *static_type,
 //   multiple paths from the same dst_type to here, record the "most public" one.
 //   Record the dst_ptr as pointing to (static_ptr, static_type).
 //   If more than one (dst_ptr, dst_type) points to (static_ptr, static_type),
-//   then mark this dyanmic_cast as ambiguous and stop the search.
+//   then mark this dynamic_cast as ambiguous and stop the search.
```

Comment:
This is an update llvm file.  Maybe revert this, and send a patch upstream to llvm if this issue is still present there?